README.md

```diff
@@ -170,6 +170,9 @@ flank:
 
   ## The max time this test run can execute before it is cancelled (default: unlimited).
   # run-timeout: 60m
+
+  ## Set true to end with exit code 0 despite of failed matrices. False by default.
```

Comment:
> ## Terminate with exit code 0 when there are failed tests. Useful for Fladle and other gradle plugins that don't expect the process to have a non-zero exit code. The JUnit XML is used to determine failure. (default: false)

What do you think of this more verbose description?

Reply:
Perfect, updated!